From d501f5186f6c03379269cd1a02aa369998da43e6 Mon Sep 17 00:00:00 2001
From: Khiem Nguyen <khiem.nguyen.xt@renesas.com>
Date: Wed, 20 Mar 2019 13:03:43 +0700
Subject: [PATCH 758/909] mfd: bd9571mwv: Clean up the BD9574MWF register
 mapping

commit a9181365a599574b10df01cabdfc7c4e1cb222c3 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch cleans up the mapped registers which is not used
during BD9574MWF PMIC operation.

Signed-off-by: Khiem Nguyen <khiem.nguyen.xt@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mfd/bd9571mwv.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/mfd/bd9571mwv.c b/drivers/mfd/bd9571mwv.c
index bb0f4ae68cc1..881e0c737f5a 100644
--- a/drivers/mfd/bd9571mwv.c
+++ b/drivers/mfd/bd9571mwv.c
@@ -122,9 +122,6 @@ static struct regmap_irq_chip bd9571mwv_irq_chip = {
 /* Regmap for BD9574MWF */
 static const struct regmap_range bd9574mwf_readable_yes_ranges[] = {
 	regmap_reg_range(BD9574MWF_VENDOR_CODE, BD9574MWF_PRODUCT_REVISION),
-	regmap_reg_range(BD9574MWF_VDCORE_VINIT, BD9574MWF_VDCORE_VINIT),
-	regmap_reg_range(BD9574MWF_VD09_VINIT, BD9574MWF_VD09_VINIT),
-	regmap_reg_range(BD9574MWF_VDCORE_SETVMAX, BD9574MWF_VDCORE_MONIVDAC),
 	regmap_reg_range(BD9574MWF_GPIO_IN, BD9574MWF_GPIO_IN),
 	regmap_reg_range(BD9574MWF_GPIO_INT, BD9574MWF_GPIO_INTMASK),
 	regmap_reg_range(BD9574MWF_GPIO_MUX, BD9574MWF_GPIO_MUX),
@@ -138,9 +135,7 @@ static const struct regmap_access_table bd9574mwf_readable_table = {
 
 static const struct regmap_range bd9574mwf_writable_yes_ranges[] = {
 	regmap_reg_range(BD9574MWF_GPIO_DIR, BD9574MWF_GPIO_OUT),
-	regmap_reg_range(BD9574MWF_VDCORE_SETVID, BD9574MWF_VDCORE_SETVID),
 	regmap_reg_range(BD9574MWF_GPIO_INT_SET, BD9574MWF_GPIO_INTMASK),
-	regmap_reg_range(BD9574MWF_GPIO_MUX, BD9574MWF_GPIO_MUX),
 	regmap_reg_range(BD9574MWF_INT_INTREQ, BD9574MWF_INT_INTMASK),
 };
 
@@ -150,7 +145,6 @@ static const struct regmap_access_table bd9574mwf_writable_table = {
 };
 
 static const struct regmap_range bd9574mwf_volatile_yes_ranges[] = {
-	regmap_reg_range(BD9574MWF_VDCORE_MONIVDAC, BD9574MWF_VDCORE_MONIVDAC),
 	regmap_reg_range(BD9574MWF_GPIO_IN, BD9574MWF_GPIO_IN),
 	regmap_reg_range(BD9574MWF_GPIO_INT, BD9574MWF_GPIO_INT),
 	regmap_reg_range(BD9574MWF_INT_INTREQ, BD9574MWF_INT_INTREQ),
-- 
2.17.1

